plugins/inputs/elasticsearch/elasticsearch.go

```diff
@@ -309,46 +316,197 @@ func (e *Elasticsearch) gatherClusterHealth(url string, acc telegraf.Accumulator
 			"unassigned_shards":     health.UnassignedShards,
 		}
 		acc.AddFields(
-			"elasticsearch_indices",
+			"elasticsearch_cluster_health_indices",
```

Comment:
I did consider this. However, this is the set of metrics for'clusterhealth' - calling it 'indices' was misleading and interfered with the naming for the new indices related information (elasticsearch_indicesstats_shards & elasticsearch_indicesstats_, elasticsearch_indicesstats_shards_primary, elasticsearch_indicesstats_shards_replica).  It seemed better to rename it to indicate its true meaning more accurately and avoid the confusion of all the other indices types of metrics.